src/Countries/Mexico.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+use Carbon\CarbonImmutable;
+
+class Mexico extends Country
+{
+    public function countryCode(): string
+    {
+        return 'mx';
+    }
+
+    /** @return array<string, CarbonImmutable> */
```

Comment:
this can be removed

Reply:
The comments right? 

For that part, done